### PR TITLE
IA-3822 wait for welder to come up as well before marking Azure runtime as Running

### PR DESCRIPTION
**Pull request by Qi77Qi**
\<your comments for this PR go here\>

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green (you can re-run automation tests with a comment saying `jenkins retest`
- [ ] Run the automation tests multiple times in parallel to weed out instability if applicable via a comment saying `multi-test`
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: http/src/test/resources/reference.conf [hunk 1]
```diff
@@ -300,7 +300,7 @@ azure {
    pubsub-handler {
         create-vm-poll-config {
            initial-delay = 1 seconds
-           max-attempts = 120
+           max-attempts = 10
            interval = 1 seconds
```
Comment (n8landolt) [inline http/src/test/resources/reference.conf:304]: Do we want to increase the interval to give the service more time to come up?

File: http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/ConfigReaderSpec.scala [hunk 2]
```diff
@@ -38,7 +38,7 @@ class ConfigReaderSpec extends AnyFlatSpec with Matchers {
           Uri.unsafeFromString("https://localhost:8000"),
           "terradevacrpublic.azurecr.io/welder-server",
           "ef956b2",
-          PollMonitorConfig(1 seconds, 120, 1 seconds),
+          PollMonitorConfig(1 seconds, 10, 1 seconds),
```
Comment (n8landolt) [inline http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/ConfigReaderSpec.scala:41]: I made this adjustment for you, test was failing.

**Review by n8landolt — APPROVED** [review]

File: http/src/test/resources/reference.conf (hunk 1 above)
Comment (Qi77Qi) [reply http/src/test/resources/reference.conf:304]: nah...this is only for unit tests where we just mock the results typically..so shorter attempts make sense

File: http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/ConfigReaderSpec.scala (hunk 2 above)
Comment (Qi77Qi) [reply http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/ConfigReaderSpec.scala:41]: ah great..thanks!

**Pull request merged**